jdaviz/configs/specviz/helper.py

```diff
@@ -189,20 +189,26 @@ def flip_y(self):
     def show(self):
         self.app
 
-    def set_redshift_slider_bounds(self, lower=None, upper=None, step=None):
+    def set_redshift_slider_bounds(self, range=None, step=None):
         '''
-        Set the upper, lower, or both bounds of the redshift slider. Note
-        that this does not do any sanity checks on the numbers provided based
-        on whether the slider is set to Redshift or Radial Velocity.
+        Set the range and/or step of the redshift slider. Set either/both to 'auto'
+        to default based on the limits of the spectrum plot.
+
+        Parameters
+        ----------
+        range
+            Specifies the difference between the upper and lower bounds of the slider.
+            Note that the slider specifies redshift delta from the current value, so a
+            range of 0.1 would allow the user to change the current redshift by +/- 0.05.
+        step
```

Comment:
What if default (`None`) is used -- what is the behavior then? Should we put in a check to make sure step will never exceed the range?
```suggestion
        step : float or `None`
```

Reply:
see 4371f63 for the change to the docstring, and 0a66924 for addition of check